gtest/src/program.rs

```diff
@@ -405,6 +405,14 @@ impl<'a> Program<'a> {
             .call_meta(&self.id, None, "meta_state")
     }
 
+    pub fn add_value(&mut self, value: Balance) {
```

Comment:
or `mint` probably?

Reply:
I was thinking about the `mint` name too, so you've confirmed that it is a more appropriate name.

Renamed `add_value` -> `mint`, `add_value_to` -> `mint_to`.